### ISSUE TITLE
node12.16.0  Operation error

### ISSUE THREAD

**Issue by ShareQiu1994** [CesiumGS/cesium#8644]
I'm using nodejs 12.16.0 and cesium 1.66

In node 12.16.0, I found that cesium couldn't run：
```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: D:\github\CesiumJS\server.js
[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:13)[39m
[90m    at Module.load (internal/modules/cjs/loader.js:1001:32)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:900:14)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)[39m
[90m    at internal/main/run_main_module.js:18:47[39m {
  code: [32m'ERR_REQUIRE_ESM'[39m
}
```
When I switch to node 10.17.0, cesium 1.66 can run

**Comment by ShareQiu1994** [comment]
Does cesium plan to support the new version of nodejs？

**Comment by mramato** [comment]
@ShareQiu1994 thanks for opening an issue.

Cesium already supports Node 12 and I am unable to reproduce this problem with 12.6.0. Can you please provide the exact steps to recreate it?

It appears Node 12.6.0 has 6 significant regressions that they quickly patched in 12.6.1: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#2020-02-18-version-12161-erbium-lts-mylesborins Perhaps you are hitting one of these?

**Comment by ShareQiu1994** [comment]
@mramato Thank you for your answer.

repro steps:
1.git clone https://github.com/CesiumGS/cesium.git
2.npm install
3.node server.js

My environment：
Nodejs 12.16.1
Cesium 1.66
Windows 10

When I upgraded the node to 12.16.1, I found that the problem still existed.
I don't know if it's a problem with my system environment. I'll try changing the machine.Thank

**Comment by mramato** [comment]
@ShareQiu1994 My apologize but I can now reproduce this.  I accidentally ran with 12.6 instead of 12.16.  This is a breaking change during the 12.x line from the NodeJS team.  Looks like they changed behavior as of 12.10.

I'm looking into it.

**Comment by mramato** [comment]
There are 2 issues here:

1. Make Cesium's dev server and release package operations all run under Node 12.  This should be easy.

2. Make Cesium itself build using Node 12.  This is harder.  It looks like the breaking changes from Node 13 were backported to 12.  There is an initial PR in #8541 but there are showstoppers in our dependencies.

CesiumJS itself works under Node 12, Node 12 based applications can import and use it without issue. It's only the developer tools we use to build Cesium that actually have issues.

**Comment by mramato** [comment]
Updated my above comment but I'm closing this issue in favor of tracking it as part of #8452

**Comment by ShareQiu1994** [comment]
@mramato Thank you for your efforts, I will continue to pay attention to its changes

**Comment by mramato** [comment]
This is fixed in master and will be released as part of 1.68 on April 1st.
